amundsen_application/static/js/components/TableDetail/ColumnListItem/index.tsx

```diff
@@ -172,19 +185,21 @@ export class ColumnListItem extends React.Component<
           {this.state.isExpanded && (
             <section className="expanded-content">
               <div className="stop-propagation" onClick={this.stopPropagation}>
-                <EditableSection
-                  title="Description"
-                  readOnly={!metadata.is_editable}
-                  editText={this.props.editText}
-                  editUrl={this.props.editUrl}
-                >
-                  <ColumnDescEditableText
-                    columnIndex={this.props.index}
-                    editable={metadata.is_editable}
-                    maxLength={getMaxLength('columnDescLength')}
-                    value={metadata.description}
-                  />
-                </EditableSection>
+                {this.shouldRenderDescription() && (
+                  <EditableSection
+                    title="Description"
```

Comment:
Could you move this string into a constant? Something like EDITABLE_SECTION_TITLE or similar.

Reply:
+1